modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/java/com/liferay/dynamic/data/mapping/web/internal/exportimport/data/handler/DDMStructureStagedModelDataHandler.java

```diff
@@ -227,8 +228,16 @@ protected void doExportStagedModel(
 		}
 
 		List<DEDataDefinitionFieldLink> deDataDefinitionFieldLinks =
-			_deDataDefinitionFieldLinkLocalService.
-				getDEDataDefinitionFieldLinks(structure.getStructureId());
+			ListUtil.concat(
+				_deDataDefinitionFieldLinkLocalService.
+					getDEDataDefinitionFieldLinksByClassNameIdAndClassPK(
+						_portal.getClassNameId(DDMStructure.class.getName()),
+						structure.getStructureId()),
+				_deDataDefinitionFieldLinkLocalService.
+					getDEDataDefinitionFieldLinksByClassNameIdAndClassPK(
+						_portal.getClassNameId(
+							DDMStructureLayout.class.getName()),
+						structure.getDefaultDDMStructureLayoutId()));
```

Comment:
@mbowerman shouldn't we consider the cases where the `className` is not `DDMStructure` and is not `DDMStructureLayout`? For example, [here](https://github.com/carolmariaabb/liferay-portal/blob/10ff0c6bd7dcd01f0d1f563d3e652530d0239d37/modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/resource/v2_0/DataListViewResourceImpl.java#L278-L280) we are adding `DEDataDefinitionFieldLinks` with `DEDataListView` as the `className`.

Reply:
Makes sense. Thanks, @mbowerman!